umap/views.py

```diff
@@ -622,6 +622,28 @@ def post(self, *args, **kwargs):
         return simple_json_response()
 
 
+class SendEditLink(FormLessEditMixin, PermissionsMixin, UpdateView):
+    model = Map
+    pk_url_kwarg = 'map_id'
+
+    def form_valid(self, form):
+        if (self.object.owner
+           or not self.object.is_anonymous_owner(self.request)
+           or not self.object.can_edit(self.request.user, self.request)):
+            return HttpResponseForbidden()
+        email = form.cleaned_data["email"]
+        from django.core.mail import send_mail
```

Comment:
Does `send_mail` import requires external dependencies/settings? If so, maybe add a comment?

Reply:
Yes, settings. I'll add an entry in the documentation.